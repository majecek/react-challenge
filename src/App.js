import React, { Component } from 'react'
import Home from './components/Home'

const style = {
  display: 'flex',
  flexDirection: 'column',
  alignItems: 'center',
  justifyContent: 'flex-start'
}

class App extends Component {
  render () {
    return (
      <div style={style}>
        <Home/>
      </div>
    )
  }
}

export default App
