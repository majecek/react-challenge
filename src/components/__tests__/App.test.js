import React from 'react'
import ReactDOM from 'react-dom'
import App from '../../App'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('App component tests', () => {
  const component = renderer.create(<App/>).toJSON()
  expect(component).toMatchSnapshot()
})