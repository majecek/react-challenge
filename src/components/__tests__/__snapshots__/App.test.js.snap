// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
        "padding": 10,
      }
    }
  >
    <div>
      <form
        className="input-group"
        onSubmit={[Function]}
      >
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="insert your array separated by comma"
          value=""
        />
        <span
          className="input-group-btn"
        >
          <button
            className="btn btn-secondary"
            type="submit"
          >
            Submit
          </button>
        </span>
      </form>
      <div
        className="form-group has-danger"
      >
        <div
          className="form-control-feedback"
        >
          
        </div>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-block text-nowrap"
      >
        Total water in sinkbase: 
        0
      </div>
    </div>
    <div
      style={
        Object {
          "padding": 20,
        }
      }
    >
      <svg
        className="Matrix"
        height={50}
        width={0}
      >
        <g>
          <g
            className="Row"
          />
        </g>
      </svg>
    </div>
  </div>
</div>
`;
